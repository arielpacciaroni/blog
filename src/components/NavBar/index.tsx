import Link from '../Link'

export default function NavBar() {
  return (
    <nav className="section-wrapper py-4 border-b border-stone-900">
      <section className="flex justify-between">
        <Link href="/">Home</Link>
        <Link href="/about">About Me</Link>
      </section>
    </nav>
  )
}
