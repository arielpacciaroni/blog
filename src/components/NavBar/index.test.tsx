import { render } from '@testing-library/react'
import NavBar from '.'
import '@testing-library/jest-dom'

describe('NavBar', () => {
  it('renders navbar unchanged', () => {
    const { container } = render(<NavBar />)
    expect(container).toMatchSnapshot()
  })
})
