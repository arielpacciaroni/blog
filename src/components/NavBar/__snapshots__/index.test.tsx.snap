// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders navbar unchanged 1`] = `
<div>
  <nav
    class="section-wrapper py-4 border-b border-stone-900"
  >
    <section
      class="flex justify-between"
    >
      <a
        class="text-gray-300 hover:text-white"
        href="/"
      >
        Home
      </a>
      <a
        class="text-gray-300 hover:text-white"
        href="/about"
      >
        About Me
      </a>
    </section>
  </nav>
</div>
`;
